### DISCUSSION TITLE
sleep problem

### DISCUSSION THREAD

**Discussion by ilker** [EssentialsX/Essentials#4118] [Q&A] [answered]
When one of us goes to sleep it turns to day even though the rest of us aren't in our beds! I want to turn this off so that all three of us are required to be in bed in order for it to become a new day.

nobody has the "essentials.sleepingignored" authority.

**Comment by pop4959** [comment, marked as answer]
It sounds like you do have the permission, or some other plugin is causing that. You can use LP verbose to figure out if you need to disable the perm or not.
